### PR TITLE
Master sync for v10.27

### PR DESCRIPTION
**Pull request by PeterYinusa**
## Explanation

This brings develop up-to-date with v10.27

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.27.0]
+### Added
+- feat: add the ConsenSys zkEVM (Linea) as a default network ([#17875](https://github.com/MetaMask/metamask-extension/pull/17875))
```
[the remark this answers is not in the document]
Comment (pedronfigueiredo) [reply CHANGELOG.md:11]: (this is because I'm not sure if the actual network will be called Linea too)

[the remark this answers is not in the document]
Comment (PeterYinusa) [reply CHANGELOG.md:11]: Agreed, the network appears as `Linea Goerli test network` in the network picker, so we should probably use consistent language in the changelog.
However, I'm not sure this PR is the best place to update it, as this PR solely brings the develop branch up to date with changes from the master branch.
Let me know if you still this change is best placed here and i'll update it

**Review by seaona — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (seaona) [reply CHANGELOG.md:11]: I would also vote for fixing this in a different PR, as this PR is intended to sync with master

[the remark this answers is not in the document]
Comment (pedronfigueiredo) [reply CHANGELOG.md:11]: Ah, that makes sense! LGTM Peter, we can see later if it makes sense to update the changelog on the next release

**Review by pedronfigueiredo — APPROVED** [review]

**Pull request merged**